nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Well done, you did an awesome job but here is the most important thing that you need to do alongside just raising or returning the `Error` to the user, you should also send a message back to the user, in case of happening something wrong.

```suggestion
   raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
```